cmd/ejson/main.go

```diff
@@ -65,9 +66,22 @@ func main() {
 					Name:  "o",
 					Usage: "print output to the provided file, rather than stdout",
 				},
+				cli.BoolTFlag{
```

Comment:
`BoolTFlag` is true by default, right? I think you'd want `BoolFlag` so that it's false by default.